HW2/task_2.py

```diff
@@ -0,0 +1,47 @@
+# Напишите программу, которая принимает две строки
+# вида “a/b” — дробь с числителем и знаменателем.
+# Программа должна возвращать сумму
+# и *произведение дробей. Для проверки своего
+# кода используйте модуль fractions.
+
+from fractions import Fraction
+
+# data_1, data_2 = input('Введите первую дробь: '), input('Введите вторую дробь: ')
+data_1 = '2/3'
+data_2 = '5/18'
```

Comment:
Задал дроби так
```suggestion
data_1 = '1/2'
data_2 = '1/2'
```
Вот каким был результат. Ничего ен совпадает
![image](https://github.com/YuBarSoft/Immersion_in_Python-HW/assets/49741194/7d46abe9-5133-4fd0-90e1-e049bb33b784)

Reply:
Потом задал так:
```suggestion
data_1 = '0/2'
data_2 = '1/2'
```
Тоже не совпадает
![image](https://github.com/YuBarSoft/Immersion_in_Python-HW/assets/49741194/18e72821-959c-4791-9979-32f6462b06e6)

Reply:
Уважаемый Данил, спасибо за указание на ошибки в решении задачи. 
Исправлено. Сейчас вроде все должно работать правильно )